### ISSUE TITLE
BasicVSR++: too hard for training

### ISSUE THREAD

**Issue by meobach** [open-mmlab/mmediting#743]
Hi,
I have joined nitre 2022 challenge and tried to retrain BasicVSR++ on their new dataset on track1. But it is too hard for improvement.
The pre-trained model last year achieved 31.64 PSNR on val set, but when I retrain it with the same config, I only got 30.0 PSNR after two days of training(the score of the first 100 iterations is: 29.83db, so for 2 days, it only increase 0.17db).  Do you have any trick here to speed up the training process?
Thank you

**Comment by ckkelvinchan** [comment]
You can try fine-tuning the pre-trained model with the new data. Actually, we rarely trained a model from scratch last year. We keep fine-tuning the models to get the final model.

**Comment by meobach** [comment]
Thank for your reply, I haved fine-tuning it, but its score slightly goes down from 0.1-0.2db after every 200 iterations, I tried to change the learning rate but it doesn't help so much. Do you have any tricks for the fine-tuning process?

**Comment by ckkelvinchan** [comment]
It is normal that PSNR and SSIM drop a bit at the beginning. You can imagine it as a learning rate restart. Normally the metrics will rise eventually.

We did not use any special tricks during fine-tuning. We just repeat the training multiple times.

**Comment by meobach** [comment]
Ok, thks for you help
